app/src/test/java/org/mozilla/vrbrowser/GleanMetricsServiceTest.kt

```diff
@@ -0,0 +1,34 @@
+package org.mozilla.vrbrowser
+
+import androidx.test.core.app.ApplicationProvider
+import mozilla.components.service.glean.Glean.initialize
+import mozilla.components.service.glean.Glean.setUploadEnabled
+import mozilla.components.service.glean.config.Configuration
+import mozilla.components.service.glean.config.Configuration.Companion.DEFAULT_TELEMETRY_ENDPOINT
+import mozilla.components.service.glean.testing.GleanTestRule
+import org.junit.Assert.assertEquals
+import org.junit.Rule
+import org.junit.Test
+import org.mozilla.vrbrowser.GleanMetrics.*
+
+
+@get:Rule
```

Comment:
As [documented here](https://mozilla.github.io/glean/book/user/testing-metrics.html#general-test-api-method-semantics), this rule should be within your `GleanMetricsServiceTest` class.